pkg/cloud/gcp/gcs_storage_test.go

```diff
@@ -99,6 +100,83 @@ func isImplicitAuthConfigured() bool {
 	return credentials != ""
 }
 
+func TestGCSAssumeRole(t *testing.T) {
```

Comment:
Can we try and get these tests running in the new nightly job we added? Happy to help configure it appropriately.

Reply:
done, got it working with the new nightly job and verified that it passes.